src/System.Text.Encoding.Extensions/tests/ASCII.cs

```diff
@@ -15,142 +15,26 @@ public static class ASCII
         [Fact]
         public static void GetByteCount_BoundaryValues()
         {
-            s_encodingUtil_ASCII.GetByteCountTest(String.Empty, 0, 0, 0);
-            Assert.Throws<ArgumentOutOfRangeException>(() => s_encodingUtil_ASCII.GetByteCountTest(String.Empty, 0, 1, 0));
-            Assert.Throws<ArgumentNullException>(() => s_encodingUtil_ASCII.GetByteCountTest((String)null, 0, 0, 0));
-            Assert.Throws<ArgumentOutOfRangeException>(() => s_encodingUtil_ASCII.GetByteCountTest("abc", -1, 0, 0));
-            Assert.Throws<ArgumentOutOfRangeException>(() => s_encodingUtil_ASCII.GetByteCountTest("abc", 0, -1, 0));
-            Assert.Throws<ArgumentOutOfRangeException>(() => s_encodingUtil_ASCII.GetByteCountTest("abc", -1, -1, 0));
-            Assert.Throws<ArgumentOutOfRangeException>(() => s_encodingUtil_ASCII.GetByteCountTest("abc", 1, -1, 0));
-            s_encodingUtil_ASCII.GetByteCountTest("abc", 3, 0, 0);
-            Assert.Throws<ArgumentOutOfRangeException>(() => s_encodingUtil_ASCII.GetByteCountTest("abc", 3, 1, 0));
             s_encodingUtil_ASCII.GetByteCountTest("abc", 2, 1, 1);
```

Comment:
in general, why we still keeping such tests? I am seeing we still keeping many of them. is that because the tests on System.Text.Encoding is not covering such cases yet? or is there any other reason?